src/components/PersonLink.tsx

```diff
@@ -0,0 +1,67 @@
+import { Link, useLocation } from 'react-router-dom';
+import { Person } from '../types';
+import classNames from 'classnames';
+import { GetPersonDates } from '../services/getPersonDates';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { name, sex, born, died, slug, motherName, fatherName } = person;
+
+  const MOTHER_NAME = motherName;
+  const FATHER_NAME = fatherName;
+
+  const { pathname } = useLocation();
+
+  const onlySlug = pathname.split('/').slice(2).join();
+
+  const mother = person.motherName ? person.motherName : '-';
+  const father = person.fatherName ? person.fatherName : '-';
+
+  const motherDates = GetPersonDates(MOTHER_NAME, 'mother');
+  const fatherDates = GetPersonDates(FATHER_NAME, 'father');
+
+  return (
+    <>
+      <tr
+        data-cy="person"
+        className={classNames('', {
+          'has-background-warning': onlySlug === slug,
+        })}
+      >
+        <td>
+          <Link
+            to={`/people/${slug}`}
+            className={classNames('', {
+              'has-text-danger': sex === 'f',
```

Comment:
```suggestion
              'has-text-danger': sex === FEMALE,
```
It's better to use constants instead of magic strings